specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/OrderCollectionCreate.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "orderCollectionName": "TestOrderCollectionName1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/orderCollections/TestOrderCollectionName1?api-version=2020-12-01-preview"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "447"
+    ],
+    "orderCollectionResource": {
+      "properties": {
+        "OrderIds": [
+          "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourcegroups/TestRG/providers/Microsoft.EdgeOrder/Orders/TestOrderName1",
+          "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourcegroups/TestRG1/providers/Microsoft.EdgeOrder/Orders/TestOrderName2"
+        ]
+      },
+      "tags": {
+        "Name": "TestName",
+        "Work": "Engineering"
+      },
+      "location": "westus",
+      "type": "Microsoft.EdgeOrder/orderCollections"
+    }
+  },
+  "responses": {
+    "202": {
+      "body": ""
+    },
+    "200": {
+      "body": {
+        "properties": {
+          "orderIds": [
+            "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/orders/TestOrderName1",
+            "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG1/providers/Microsoft.EdgeOrder/orders/TestOrderName2"
```

Comment:
how will you handle order deletes? will you remove this entry from the corresponding order collection too?

Reply:
Yes